### PR TITLE
feat(client): upgrade link updates. 

### PR DESCRIPTION
**Pull request by morhag90**
Close: https://github.com/amplication/private-issues/issues/172

## PR Details

- In the featureIndicator tooltip, change the "upgrade" link to "talk to us" and the route to set a meeting. 
- Update the message content. 

## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

### CODE REVIEW

File: packages/amplication-client/src/Components/FeatureIndicator.tsx [hunk 1]
```diff
@@ -32,9 +32,10 @@ const WarningTooltip = styled(
 
 const CLASS_NAME = "amp-feature-indicator";
 
-export const tooltipDefaultTextUpgrade = "Upgrade";
-
-export const tooltipDefaultText = `Explore this feature, included in your 7-day Enterprise trial. ${tooltipDefaultTextUpgrade} for continued access.`;
+export const defaultTextStart =
+  "Explore this feature, included in your 7-day Enterprise trial.";
+export const defaultTextEnd =
+  "today to ensure continued access and discover additional hidden functionalities!";
```
Comment (abrl91) [inline packages/amplication-client/src/Components/FeatureIndicator.tsx:38]:
```suggestion
  "Today to ensure continued access and discover additional hidden functionalities!";
```

File: packages/amplication-client/src/Components/LicenseIndicatorContainer.tsx [hunk 2]
```diff
@@ -112,12 +112,12 @@ export const LicenseIndicatorContainer: FC<Props> = ({
...
 
-  const linkText = useMemo(() => {
+  const showTooltipLink = useMemo(() => {
     if (isPreviewPlan(subscriptionPlan)) {
-      return ""; // don't show the upgrade link when the plan is preview
+      return false; // don't show the upgrade link when the plan is preview
     }
 
-    return undefined; // in case of null, it falls back to the default link text
+    return true; // in case of null, it falls back to the default link text
```
Comment (abrl91) [inline packages/amplication-client/src/Components/LicenseIndicatorContainer.tsx:120]: Please check if the comment is still relevant or needs to be updated

File: packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx [hunk 3]
```diff
@@ -150,15 +150,15 @@ const WorkspaceHeader: React.FC = () => {
   );
 
   const handleUpgradeClick = useCallback(() => {
-    history.push(`/${currentWorkspace.id}/purchase`, {
-      from: { pathname: window.location.pathname },
-    });
+    // history.push(`/${currentWorkspace.id}/purchase`, {
+    //   from: { pathname: window.location.pathname },
+    // });
```
Comment (abrl91) [inline packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx:155]: remove

File: packages/amplication-client/src/Components/FeatureIndicator.tsx (hunk 1 above)
Comment (morhag90) [reply packages/amplication-client/src/Components/FeatureIndicator.tsx:38]: @abrl91, no need, this text coming after the dynamic text: "Talk to us"/ "Talk with our CEO"

**Review by abrl91 — APPROVED** [review]

**Review by abrl91 — APPROVED** [review]

**Pull request merged**